pypeit/inputfiles.py

```diff
@@ -391,18 +463,26 @@ def path_and_files(self, key:str, skip_blank=False, check_exists=True):
             if skip_blank and row[key].strip() in ['', 'none', 'None']:
                 continue
 
+            # Skip commented out entries
+            if row[key].strip().startswith("#"):
+                if not include_commented_out:
+                    continue
+                name = row[key].strip("# ")
```

Comment:
Might be worth adding an in-line comment here that the space after the hash is important.  I.e., `"# "` vs. `"#"` is an important difference.